<html><head><title>EC 3.4.22.55</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC3/4/22/55.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 3.4.22.55</b></h1></center><b>Accepted name:</b> caspase-2<p><b>Reaction:</b> Strict requirement for an Asp residue at P1, with Asp<small><sup>316</sup></small> being essential for proteolytic activity and has a preferred cleavage sequence of  Val-Asp-Val-Ala-Asp<img src="../../EZgif/FISS.GIF" align=top><p><b>Other name(s):</b> ICH-1; NEDD-2; caspase-2L; caspase-2S; neural precursor cell expressed developmentally down-regulated protein 2; CASP-2; NEDD2 protein<p><b>Comments:</b> Caspase-2 is an initiator caspase, as are caspase-8 (<a href="61.html">EC 3.4.22.61</a>), caspase-9 (<a href="62.html">EC 3.4.22.62</a>) and caspase-10 (<a href="64.html">EC 3.4.22.64</a>) [6]. Contains a caspase-recruitment domain (CARD) in its N-terminal prodomain, which plays a role in procaspase activation [6]. Two forms of caspase-2 with antagonistic effects exist: caspase-2L induces programmed cell death and caspase-2S suppresses cell death [2,3,5]. Caspase-2 is activated by caspase-3 (<a href="56.html">EC 3.4.22.56</a>), or by a caspase-3-like protease. Activation involves cleavage of the N-terminal prodomain, followed by self-proteolysis between the large and small subunits of pro-caspase-2 and further proteolysis into smaller fragments [3]. Proteolysis occurs at Asp residues and the preferred substrate for this enzyme is a pentapeptide rather than a tetrapeptide [5]. Apart from itself, the enzyme can cleave golgin-16, which is present in the Golgi complex and has a cleavage site that is unique for caspase-2 [4,5]. &alpha;II-Spectrin, a component of the membrane cytoskeleton, is a substrate of the large isoform of pro-caspase-2 (caspase-2L) but not of the short isoform (caspase-2S). Belongs in <a href="http://merops.sanger.ac.uk/cgi-bin/merops.cgi?id=C14" target="new">peptidase family C14</a>.<p><b>Links to other databases</b>: <a target="new" href="http://www.brenda-enzymes.org/enzyme.php?ecno=3.4.22.55">BRENDA</a>, <a target="new" href="http://enzyme.expasy.org/EC/3.4.22.55">EXPASY</a>, <a target="new" href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:3.4.22.55">KEGG</a>, <a target="new" href="http://merops.sanger.ac.uk/cgi-bin/merops.cgi?id=C14.006">MEROPS</a>, <a href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-3.4.22.55" target="new">Metacyc</a>,<a target="new" href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=3.4.22.55">PDB</a>, CAS registry number: 182372-14-1<p><b>References:</b><p>1. Kumar, S., Kinoshita, M., Noda, M., Copeland, N.G. and Jenkins, N.A. Induction of apoptosis by the mouse Nedd2 gene, which encodes a protein similar to the product of the <i>Caenorhabditis elegans</i> cell death gene <i>ced-3</i>  and the mammalian IL-1&beta;-converting enzyme. <i>Genes Dev.</i> 8 (1994) 1613-1626. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/7958843?dopt=Abstract" target="new">7958843</a>]<p>2. Wang, L., Miura, M., Bergeron, L., Zhu, H. and Yuan, J. <i>Ich-1</i>, an <i>Ice/ced-3</i>-related gene, encodes both positive and negative regulators of programmed cell death. <i>Cell</i> 78 (1994) 739-750. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/8087842?dopt=Abstract" target="new">8087842</a>]<p>3. Li, H., Bergeron, L., Cryns, V., Pasternack, M.S., Zhu, H., Shi, L., Greenberg, A. and Yuan, J. Activation of caspase-2 in apoptosis. <i>J. Biol. Chem.</i> 272 (1997) 21010-21017. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/9261102?dopt=Abstract" target="new">9261102</a>]<p>4. Mancini, M., Machamer, C.E., Roy, S., Nicholson, D.W., Thornberry, N.A., Casciola-Rosen, L.A. and Rosen, A. Caspase-2 is localized at the Golgi complex and cleaves golgin-160 during apoptosis. <i>J. Cell Biol.</i> 149 (2000) 603-612. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/10791974?dopt=Abstract" target="new">10791974</a>]<p>5. Zhivotovsky, B. and Orrenius, S. Caspase-2 function in response to DNA damage. <i>Biochem. Biophys. Res. Commun.</i> 331 (2005) 859-867. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/15865942?dopt=Abstract" target="new">15865942</a>]<p>6. Chang, H.Y. and Yang, X. Proteases for cell suicide: functions and regulation of caspases. <i>Microbiol. Mol. Biol. Rev.</i> 64 (2000) 821-846. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/11104820?dopt=Abstract" target="new">11104820</a>]<p><center><small>[EC 3.4.22.55 created 2006]</small></center><p><hr>Return to <a href="../22/">EC 3.4.22 home page</a><br>Return to <a href="../">EC 3.4 home page</a><br>Return to <a href="../../">EC 3 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>