<html><head><title>EC 1.14.14.33</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC1/14/14/33.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 1.14.14.33</b></h1></center><b>Accepted name:</b> ethylenediaminetetraacetate monooxygenase<p><b>Reaction:</b> ethylenediaminetetraacetate + <b>2</b> FMNH<small><sub>2</sub></small> + <b>2</b> O<small><sub>2</sub></small> = ethylenediamine-<i>N</i>,<i>N</i>'-diacetate + <b>2</b> glyoxylate + <b>2</b> FMN + <b>2</b> H<small><sub>2</sub></small>O (overall reaction)<br>(1a) ethylenediaminetetraacetate + FMNH<small><sub>2</sub></small> + O<small><sub>2</sub></small> = ethylenediaminetriacetate + glyoxylate + FMN + H<small><sub>2</sub></small>O<br>(1b) ethylenediaminetriacetate + FMNH<small><sub>2</sub></small> + O<small><sub>2</sub></small> = ethylenediamine-<i>N</i>,<i>N</i>'-diacetate + glyoxylate + FMN + H<small><sub>2</sub></small>O<p><b>Glossary:</b> ethylenediaminetetraacetate = EDTA<p><b>Systematic name:</b> ethylenediaminetetraacetate,FMNH<small><sub>2</sub></small>:O<small><sub>2</sub></small> oxidoreductase (glyoxylate-forming)<p><b>Comments:</b> The enzyme is part of a two component system that also includes <a href="../../5/1/42.html">EC 1.5.1.42</a>, FMN reductase (NADH), which provides reduced flavin mononucleotide for this enzyme. It acts on EDTA only when it is complexed with divalent cations such as Mg<small><sup>2+</sup></small>, Zn<small><sup>2+</sup></small>, Mn<small><sup>2+</sup></small>, Co<small><sup>2+</sup></small>, or Cu<small><sup>2+</sup></small>. While the enzyme has a substrate overlap with <a href="10.html">EC 1.14.14.10</a>, nitrilotriacetate monooxygenase, it has a much wider substrate range, which includes nitrilotriacetate (NTA) and diethylenetriaminepentaacetate (DTPA) in addition to EDTA.<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.info/php/result_flat.php3?ecno=1.14.14.33" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/1.14.14.33" target="new">EXPASY</a>,<a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:1.14.14.33" target="new">KEGG</a>, <a href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-1.14.14.33" target="new">Metacyc</a>,CAS registry number: <p><b>References:</b><p>1. Witschel, M., Nagel, S. and Egli, T. Identification and characterization of the two-enzyme system catalyzing oxidation of EDTA in the EDTA-degrading bacterial strain DSM 9103. <i>J. Bacteriol.</i> 179 (1997) 6937-6943. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/9371437?dopt=Abstract" target="new">9371437</a>]<p>2. Payne, J.W., Bolton, H., Jr., Campbell, J.A. and Xun, L. Purification and characterization of EDTA monooxygenase from the EDTA-degrading bacterium BNC1. <i>J. Bacteriol.</i> 180 (1998) 3823-3827. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/9683478?dopt=Abstract" target="new">9683478</a>]<p>3. Bohuslavek, J., Payne, J.W., Liu, Y., Bolton, H., Jr. and Xun, L. Cloning, sequencing, and characterization of a gene cluster involved in EDTA degradation from the bacterium BNC1. <i>Appl. Environ. Microbiol.</i> 67 (2001) 688-695. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11157232?dopt=Abstract" target="new">11157232</a>]<p><center><small>[EC 1.14.14.33 created 2016]</small></center><p><hr>Return to <a href="../14/">EC 1.14.14 home page</a><br>Return to <a href="../">EC 1.14 home page</a><br>Return to <a href="../../">EC 1 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>