<html><head><title>EC 3.1.26.13</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC3/1/26/13.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 3.1.26.13</b></h1></center><b>Accepted name:</b> retroviral ribonuclease H<p><b>Reaction:</b>  Endohydrolysis of RNA in RNA/DNA hybrids. Three different cleavage modes:<br>1. sequence-specific internal cleavage of RNA [1-4]. Human immunodeficiency virus type 1 and Moloney murine leukemia virus enzymes prefer to cleave the RNA strand one nucleotide away from the RNA-DNA junction [5].<br>2. RNA 5'-end directed cleavage 13-19 nucleotides from the RNA end [6,7].<br>3. DNA 3'-end directed cleavage 15-20 nucleotides away from the primer terminus [8-10]. <p><b>Other name(s):</b> RT/RNase H; retroviral reverse transcriptase RNaseH; HIV RNase H<p><b>Comments:</b> Comments: Retroviral reverse transcriptase is a multifunctional enzyme responsible for viral replication. To perform this task the enzyme combines two distinct activities. The polymerase domain (<a href="../../../EC2/7/7/49.html">EC 2.7.7.49</a>, RNA-directed DNA polymerase) occupies the N-terminal two-thirds of the reverse transcriptase whereas the ribonuclease H domain comprises the C-terminal remaining one-third [13,14]. The RNase H domain of Moloney murine leukemia virus and Human immunodeficiency virus display two metal binding sites [15-17]<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=3.1.26.13" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/3.1.26.13" target="new">EXPASY</a>, <a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:3.1.26.13" target="new">KEGG</a>, <a target="new" href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-3.1.26.13">Metacyc</a>,<a href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=3.1.26.13" target="new">PDB</a>, CAS registry number: <p><b>References:</b><p>1. Schultz, S.J., Zhang, M. and Champoux, J.J. Recognition of internal cleavage sites by retroviral RNases H. <i>J. Mol. Biol.</i> 344 (2004) 635-652. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15533434?dopt=Abstract" target="new">15533434</a>]<p>2. Sarafianos, S.G., Das, K., Tantillo, C., Clark, A.D., Jr., Ding, J., Whitcomb, J.M., Boyer, P.L., Hughes, S.H. and Arnold, E. Crystal structure of HIV-1 reverse transcriptase in complex with a polypurine tract RNA:DNA. <i>EMBO J.</i> 20 (2001) 1449-1461. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11250910?dopt=Abstract" target="new">11250910</a>]<p>3. Rausch, J.W., Lener, D., Miller, J.T., Julias, J.G., Hughes, S.H. and Le Grice, S.F. Altering the RNase H primer grip of human immunodeficiency virus reverse transcriptase modifies cleavage specificity. <i>Biochemistry</i> 41 (2002) 4856-4865. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11939780?dopt=Abstract" target="new">11939780</a>]<p>4. Brehm, J.H., Mellors, J.W. and Sluis-Cremer, N. Mechanism by which a glutamine to leucine substitution at residue 509 in the ribonuclease H domain of HIV-1 reverse transcriptase confers zidovudine resistance. <i>Biochemistry</i> 47 (2008) 14020-14027. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19067547?dopt=Abstract" target="new">19067547</a>]<p>5. Schultz, S.J., Zhang, M., Kelleher, C.D. and Champoux, J.J. Analysis of plus-strand primer selection, removal, and reutilization by retroviral reverse transcriptases. <i>J. Biol. Chem.</i> 275 (2000) 32299-32309. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10913435?dopt=Abstract" target="new">10913435</a>]<p>6. DeStefano, J.J., Mallaber, L.M., Fay, P.J. and Bambara, R.A. Determinants of the RNase H cleavage specificity of human immunodeficiency virus reverse transcriptase. <i>Nucleic Acids Res.</i> 21 (1993) 4330-4338. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7692401?dopt=Abstract" target="new">7692401</a>]<p>7. Kati, W.M., Johnson, K.A., Jerva, L.F. and Anderson, K.S. Mechanism and fidelity of HIV reverse transcriptase. <i>J. Biol. Chem.</i> 267 (1992) 25988-25997. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1281479?dopt=Abstract" target="new">1281479</a>]<p>8. Palaniappan, C., Fuentes, G.M., Rodriguez-Rodriguez, L., Fay, P.J. and Bambara, R.A. Helix structure and ends of RNA/DNA hybrids direct the cleavage specificity of HIV-1 reverse transcriptase RNase H. <i>J. Biol. Chem.</i> 271 (1996) 2063-2070. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/8567660?dopt=Abstract" target="new">8567660</a>]<p>9. Fu, T.B. and Taylor, J. When retroviral reverse transcriptases reach the end of their RNA templates. <i>J. Virol.</i> 66 (1992) 4271-4278. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1376369?dopt=Abstract" target="new">1376369</a>]<p>10. Beilhartz, G.L., Wendeler, M., Baichoo, N., Rausch, J., Le Grice, S. and Gotte, M. HIV-1 reverse transcriptase can simultaneously engage its DNA/RNA substrate at both DNA polymerase and RNase H active sites: implications  for RNase H inhibition. <i>J. Mol. Biol.</i> 388 (2009) 462-474. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19289131?dopt=Abstract" target="new">19289131</a>]<p>11. Huang, H., Chopra, R., Verdine, G.L. and Harrison, S.C. Structure of a covalently trapped catalytic complex of HIV-1 reverse transcriptase: implications for drug resistance. <i>Science</i> 282 (1998) 1669-1675. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/9831551?dopt=Abstract" target="new">9831551</a>]<p>12. Krug, M.S. and Berger, S.L. Ribonuclease H activities associated with viral reverse transcriptases are endonucleases. <i>Proc. Natl. Acad. Sci. USA</i> 86 (1989) 3539-3543. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/2471188?dopt=Abstract" target="new">2471188</a>]<p>13. Champoux, J.J. and Schultz, S.J. Ribonuclease H: properties, substrate specificity and roles in retroviral reverse transcription. <i>FEBS J.</i> 276 (2009) 1506-1516. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19228195?dopt=Abstract" target="new">19228195</a>]<p>14. Schultz, S.J. and Champoux, J.J. RNase H activity: structure, specificity, and function in reverse transcription. <i>Virus Res</i> 134 (2008) 86-103. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/18261820?dopt=Abstract" target="new">18261820</a>]<p>15. Goedken, E.R. and Marqusee, S. Metal binding and activation of the ribonuclease H domain from moloney murine leukemia virus. <i>Protein Eng.</i> 12 (1999) 975-980. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10585503?dopt=Abstract" target="new">10585503</a>]<p>16. Davies, J.F., 2nd, Hostomska, Z., Hostomsky, Z., Jordan, S.R. and Matthews, D.A. Crystal structure of the ribonuclease H domain of HIV-1 reverse transcriptase. <i>Science</i> 252 (1991) 88-95. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1707186?dopt=Abstract" target="new">1707186</a>]<p>17. Pari, K., Mueller, G.A., DeRose, E.F., Kirby, T.W. and London, R.E. Solution structure of the RNase H domain of the HIV-1 reverse transcriptase in the presence of magnesium. <i>Biochemistry</i> 42 (2003) 639-650. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12534276?dopt=Abstract" target="new">12534276</a>]<p><center><small>[EC 3.1.26.13 created 2009]</small></center><p><hr>Return to <a href="../26/">EC 3.1.26 home page</a><br>Return to <a href="../">EC 3.1 home page</a><br>Return to <a href="../../">EC 3 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>