<html><head><title>EC 6.3.1.14</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC6/3/1/14.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 6.3.1.14</b></h1></center><b>Accepted name:</b> diphthine&#151;ammonia ligase<p><b>Reaction:</b> ATP + diphthine-[translation elongation factor 2] + NH<small><sub>3</sub></small> = AMP + diphosphate + diphthamide-[translation elongation factor 2]<p>For diagram of reaction <a href="../../../reaction/AminoAcid/diphthine.html">click here</a>.<p><b>Glossary:</b> translation elongation factor 2 = EF2 = eEF2<br>diphthine = 2-[(3<i>S</i>)-3-carboxy-3-(trimethylammonio)propyl]-<small>L</small>-histidine<br>diphthamide =2-[(3<i>S</i>)-3-carbamoyl-3-(trimethylammonio)propyl]-<small>L</small>-histidine<p><b>Other name(s):</b> diphthamide synthase; diphthamide synthetase; DPH6 (gene name); ATPBD4 (gene name)<p><b>Systematic name:</b> diphthine-[translation elongation factor 2]:ammonia ligase (AMP-forming)<p><b>Comments:</b> This amidase catalyses the last step in the conversion of an <small>L</small>-histidine residue in the translation elongation factor EF2 to diphthamide. This factor is found in all archaebacteria and eukaryotes, but not in eubacteria, and is the target of bacterial toxins such as the diphtheria toxin and the <i>Pseudomonas</i> exotoxin A (see <a href="../../../EC2/4/2/36.html">EC 2.4.2.36</a>, NAD<small><sup>+</sup></small>&mdash;diphthamide ADP-ribosyltransferase). The substrate of the enzyme, diphthine, is produced by <a href="../../../EC2/1/1/98.html">EC 2.1.1.98</a>, diphthine synthase.<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=6.3.1.14" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/6.3.1.14" target="new">EXPASY</a>, <a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:6.3.1.14" target="new">KEGG</a>, <a target="new" href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-6.3.1.14">Metacyc</a>,<a target="new" href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=6.3.1.14">PDB</a>, CAS registry number: 114514-33-9<p><b>References:</b><p>1. Moehring, T.J. and Moehring, J.M. Mutant cultured cells used to study the synthesis of diphthamide. <i>UCLA Symp. Mol. Cell. Biol. New Ser.</i> 45 (1987) 53-63.<p>2. Moehring, J.M. and Moehring, T.J. The post-translational trimethylation of diphthamide studied in vitro. <i>J. Biol. Chem.</i> 263 (1988) 3840-3844. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3346227?dopt=Abstract" target="new">3346227</a>]<p>3. Su, X., Lin, Z., Chen, W., Jiang, H., Zhang, S. and Lin, H. Chemogenomic approach identified yeast YLR143W as diphthamide synthetase. <i>Proc. Natl. Acad. Sci. USA</i> 109 (2012) 19983-19987. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23169644?dopt=Abstract" target="new">23169644</a>]<p><center><small>[EC 6.3.1.14 created 1990 as EC 6.3.2.22, transferred 2010 to EC 6.3.1.14, modified 2013]</small></center><p><hr>Return to <a href="../1/">EC 6.3.1 home page</a><br>Return to <a href="../">EC 6.3 home page</a><br>Return to <a href="../../">EC 6 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>