<html><head><title>EC 5.3.1.14</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC5/3/1/14.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 5.3.1.14</b></h1></center><b>Accepted name:</b> <small>L</small>-rhamnose isomerase<p><b>Reaction:</b> <small>L</small>-rhamnose = <small>L</small>-rhamnulose<p>For diagram of reaction <a href="../../../reaction/polysacc/RhaCat.html">click here</a>.<p><b>Other name(s):</b> rhamnose isomerase; <small>L</small>-rhamnose ketol-isomerase<p><b>Systematic name:</b> <small>L</small>-rhamnose aldose-ketose-isomerase<p><b>Comments:</b> Contains two divalent metal ions located at different metal-binding sites within the active site. The enzyme binds the closed ring form of the substrate and catalyses ring opening to generate a form of open-chain conformation that is coordinated to one of the metal sites. Isomerization proceeds via a hydride-shift mechanism. While the enzyme from the bacterium <i>Escherichia coli</i> is specific for <small>L</small>-rhamnose, the enzyme from the bacterium <i>Pseudomonas stutzeri</i> has broad substrate specificity and catalyses the interconversion of <small>L</small>-mannose and <small>L</small>-fructose, <small>L</small>-lyxose and <small>L</small>-xylulose, <small>D</small>-ribose and <small>D</small>-ribulose, and <small>D</small>-allose and <small>D</small>-psicose [2].<p><b>Links to other databases</b>:<a target="new" href="http://www.brenda-enzymes.org/enzyme.php?ecno=5.3.1.14">BRENDA</a>, <a target="new" href="http://enzyme.expasy.org/EC/5.3.1.14">EXPASY</a>, <a target="new" href="http://xpdb.nist.gov/enzyme_thermodynamics/enzyme_compose_query.pl?EC=5.3.1.14">GTD</a>, <a target="new" href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:5.3.1.14">KEGG</a>, <a href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-5.3.1.14" target="new">Metacyc</a>,<a target="new" href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=5.3.1.14">PDB</a>, CAS registry number: 9023-84-1<p><b>References:</b><p>1. Domagk, G.F. and Zech, R. &Uuml;ber den Abbau der Desoxyzucker durch Bakterienenzyme. I. <small>L</small>-Rhamnose-Isomerase aus <i>Lactobacillus plantarum. Biochem. Z.</i> 339 (1963) 145-153.<p>2. Leang, K., Takada, G., Ishimura, A., Okita, M. and Izumori, K. Cloning, nucleotide sequence, and overexpression of the <small>L</small>-rhamnose isomerase gene from <i>Pseudomonas stutzeri</i> in <i>Escherichia coli</i>. <i>Appl. Environ. Microbiol.</i> 70 (2004) 3298-3304. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15184124?dopt=Abstract" target="new">15184124</a>]<p>3. Korndorfer, I.P., Fessner, W.D. and Matthews, B.W. The structure of rhamnose isomerase from <i>Escherichia coli</i> and its relation with xylose isomerase illustrates a change between inter and intra-subunit complementation during evolution. <i>J. Mol. Biol.</i> 300 (2000) 917-933. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10891278?dopt=Abstract" target="new">10891278</a>]<p>4. Yoshida, H., Yamada, M., Ohyama, Y., Takada, G., Izumori, K. and Kamitori, S. The structures of <small>L</small>-rhamnose isomerase from <i>Pseudomonas stutzeri</i> in complexes with <small>L</small>-rhamnose and <small>D</small>-allose provide insights into broad substrate specificity. <i>J. Mol. Biol.</i> 365 (2007) 1505-1516. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17141803?dopt=Abstract" target="new">17141803</a>]<p><center><small>[EC 5.3.1.14 created 1965]</small></center><p><hr>Return to <a href="../1/">EC 5.3.1 home page</a><br>Return to <a href="../">EC 5.3 home page</a><br>Return to <a href="../../">EC 5 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>