<html><head><title>EC 3.4.21.116</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC3/4/21/116.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 3.4.21.116</b></h1></center><b>Accepted name:</b> SpoIVB peptidase<p><b>Reaction:</b> Self-cleaves Val<small><sup>52</sup></small><img src="../../../EZgif/FISS.GIF" align=top>Asn<small><sup>53</sup></small>, Ala<small><sup>62</sup></small><img src="../../../EZgif/FISS.GIF" align=top>Phe<small><sup>63</sup></small> and Val<small><sup>74</sup></small><img src="../../../EZgif/FISS.GIF" align=top>Thr<small><sup>75</sup></small> at the N-terminus of SpoIVB<p><b>Other name(s):</b> sporulation factor IV B protease<p><b>Comments:</b> This enzyme plays a central role in a regulatory checkpoint (the &sigma;<small><sup>K</sup></small> checkpoint), which coordinates gene expression during the later stages of spore formation in <i>Bacillus subtilis</i> [1,3]. The enzyme activates proteolytic processing of a sporulation-specific sigma factor, pro-&sigma;<small><sup>K</sup></small>, to its mature and active form, &sigma;<small><sup>K</sup></small>, by self-cleavage [1,3]. The enzyme is also subject to secondary proteolysis, which presumably inactivates SpoIVB [3]. The enzyme is also essential for the formation of heat-resistant spores. Belongs in peptidase family <a href="http://merops.sanger.ac.uk/cgi-bin/merops.cgi?id=S55 " target="new">S55</a>.<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=3.4.21.116" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/3.4.21.116" target="new">EXPASY</a>, <a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:3.4.21.116" target="new">KEGG</a>,  <a target = "new" href="http://merops.sanger.ac.uk/cgi-bin/merops.cgi?id=S55.001">MEROPS</a>, <a href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-3.4.21.116" target="new">Metacyc</a>,CAS registry number: 296241-18-4<p><b>References:</b><p>1. Wakeley, P.R., Dorazi, R., Hoa, N.T., Bowyer, J.R. and Cutting, S.M. Proteolysis of SpolVB is a critical determinant in signalling of pro-&sigma;<small><sup>K</sup></small> processing in <i>Bacillus subtilis</i>. <i>Mol. Microbiol.</i> 36 (2000) 1336-1348. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/10931284?dopt=Abstract" target="new">10931284</a>]<p>2. Hoa, N.T., Brannigan, J.A. and Cutting, S.M. The PDZ domain of the SpoIVB serine peptidase facilitates multiple functions. <i>J. Bacteriol.</i> 183 (2001) 4364-4373. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/11418578?dopt=Abstract" target="new">11418578</a>]<p>3. Hoa, N.T., Brannigan, J.A. and Cutting, S.M. The <i>Bacillus subtilis</i> signaling protein SpoIVB defines a new family of serine peptidases. <i>J. Bacteriol.</i> 184 (2002) 191-199. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/11741860?dopt=Abstract" target="new">11741860</a>]<p>4. Dong, T.C. and Cutting, S.M. SpoIVB-mediated cleavage of SpoIVFA could provide the intercellular signal to activate processing of pro-&sigma;<small><sup>K</sup></small> in <i>Bacillus subtilis. Mol. Microbiol.</i> 49 (2003) 1425-1434. [PMID: <a href="http://www.ncbi.nlm.nih.gov:80/pubmed/12940997?dopt=Abstract" target="new">12940997</a>]<p><center><small>[EC 3.4.21.116 created 2006]</small></center><p><hr>Return to <a href="../21/">EC 3.4.21 home page</a><br>Return to <a href="../">EC 3.4 home page</a><br>Return to <a href="../../">EC 3 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>