<html><head><title>EC 3.5.1.119</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC3/5/1/119.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 3.5.1.119</b></h1></center><b>Accepted name:</b> Pup amidohydrolase<p><b>Reaction:</b>  [prokaryotic ubiquitin-like protein]-<small>L</small>-glutamine + H<small><sub>2</sub></small>O = [prokaryotic ubiquitin-like protein]-<small>L</small>-glutamate + NH<small><sub>3</sub></small>  <p><b>Other name(s):</b> <i>dop</i> (gene name); Pup deamidase; depupylase/deamidase; DPUP; depupylase<p><b>Systematic name:</b> [prokaryotic ubiquitin-like protein]-<small>L</small>-glutamine amidohydrolase<p><b>Comments:</b> The enzyme has been characterized from the bacterium <i>Mycobacterium tuberculosis</i>. It catalyses the hydrolysis of the amido group of the C-terminal glutamine of prokaryotic ubiquitin-like protein (Pup), thus activating it for ligation to target proteins, a process catalysed by <a href="../../../EC6/3/1/19.html">EC 6.3.1.19</a>, prokaryotic ubiquitin-like protein ligase. The reaction requires ATP as cofactor but not its hydrolysis.The enzyme also catalyses the hydrolytic cleavage of the bond formed by the ligase, between an &epsilon;-amino group of a lysine residue of the target protein and the &gamma;-carboxylate of the C-terminal glutamate of the prokaryotic ubiquitin-like protein. <p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=3.5.1.119" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/3.5.1.119" target="new">EXPASY</a>, <a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:3.5.1.119" target="new">KEGG</a>, <a href="http://biocyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-3.5.1.119" target="new">MetaCyc</a>, CAS registry number: <p><b>References:</b><p>1. Striebel, F., Imkamp, F., Sutter, M., Steiner, M., Mamedov, A. and Weber-Ban, E. Bacterial ubiquitin-like modifier Pup is deamidated and conjugated to substrates by distinct but homologous enzymes. <i>Nat. Struct. Mol. Biol.</i> 16 (2009) 647-651. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19448618?dopt=Abstract" target="new">19448618</a>]<p>2. Burns, K.E., Cerda-Maira, F.A., Wang, T., Li, H., Bishai, W.R. and Darwin, K.H. "Depupylation" of prokaryotic ubiquitin-like protein from mycobacterial proteasome substrates. <i>Mol. Cell</i> 39 (2010) 821-827. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20705495?dopt=Abstract" target="new">20705495</a>]<p>3. Striebel, F., Imkamp, F., &Ouml;zcelik, D. and Weber-Ban, E. Pupylation as a signal for proteasomal degradation in bacteria. <i>Biochim. Biophys. Acta</i> 1843 (2014) 103-113. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23557784?dopt=Abstract" target="new">23557784</a>]<p><center><small>[EC 3.5.1.119 created 2015]</small></center><p><hr>Return to <a href="../1/">EC 3.5.1 home page</a><br>Return to <a href="../">EC 3.5 home page</a><br>Return to <a href="../../">EC 3 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>