<html><head><title>EC 2.1.1.182</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC2/1/1/182.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 2.1.1.182</b></h1></center><b>Accepted name:</b> 16S rRNA (adenine<small><sup>1518</sup></small>-<i>N</i><small><sup>6</sup></small>/adenine<small><sup>1519</sup></small>-<i>N</i><small><sup>6</sup></small>)-dimethyltransferase<p><b>Reaction:</b> <b>4</b> <i>S</i>-adenosyl-<small>L</small>-methionine + adenine<small><sup>1518</sup></small>/adenine<small><sup>1519</sup></small> in 16S rRNA = <b>4</b> <i>S</i>-adenosyl-<small>L</small>-homocysteine + <i>N</i><small><sup>6</sup></small>-dimethyladenine<small><sup>1518</sup></small>/<i>N</i><small><sup>6</sup></small>-dimethyladenine<small><sup>1519</sup></small> in 16S rRNA <p><b>Other name(s):</b> <i>S</i>-adenosylmethionine-6-<i>N</i>',<i>N</i>'-adenosyl (rRNA) dimethyltransferase; KsgA; <i>ksgA</i> methyltransferase<p><b>Systematic name:</b> <i>S</i>-adenosyl-<small>L</small>-methionine:16S rRNA (adenine<small><sup>1518</sup></small>-<i>N</i><small><sup>6</sup></small>/adenine<small><sup>1519</sup></small>-<i>N</i><small><sup>6</sup></small>)-dimethyltransferase<p><b>Comments:</b> KsgA introduces the most highly conserved ribosomal RNA modification, the dimethylation of adenine<small><sup>1518</sup></small> and adenine<small><sup>1519</sup></small> in 16S rRNA. Strains lacking the methylase are resistant to kasugamycin [1].<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=2.1.1.182" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/2.1.1.182" target="new">EXPASY</a>, <a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:2.1.1.182" target="new">KEGG</a>, <a target="new" href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-2.1.1.182">Metacyc</a>,<a href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=2.1.1.182" target="new">PDB</a>, CAS registry number: <p><b>References:</b><p>1. Helser, T.L., Davies, J.E. and Dahlberg, J.E. Change in methylation of 16S ribosomal RNA associated with mutation to kasugamycin resistance in <i>Escherichia coli. Nat. New Biol.</i> 233 (1971) 12-14. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4329247?dopt=Abstract" target="new">4329247</a>]<p>2. Helser, T.L., Davies, J.E. and Dahlberg, J.E. Mechanism of kasugamycin resistance in <i>Escherichia coli. Nat. New Biol.</i> 235 (1972) 6-9. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4336392?dopt=Abstract" target="new">4336392</a>]<p>3. van Buul, C.P. and van Knippenberg, P.H. Nucleotide sequence of the <i>ksgA</i> gene of <i>Escherichia coli</i>: comparison of methyltransferases effecting dimethylation of adenosine in ribosomal RNA. <i>Gene</i> 38 (1985) 65-72. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3905517?dopt=Abstract" target="new">3905517</a>]<p>4. Formenoy, L.J., Cunningham, P.R., Nurse, K., Pleij, C.W. and Ofengand, J. Methylation of the conserved A<small><sup>1518</sup></small>-A<small><sup>1519</sup></small> in <i>Escherichia coli</i> 16S ribosomal RNA by the <i>ksgA</i> methyltransferase is influenced by methylations around the  similarly conserved U<small><sup>1512</sup></small>.G<small><sup>1523</sup></small> base pair in the 3' terminal hairpin. <i>Biochimie</i> 76 (1994) 1123-1128. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7538324?dopt=Abstract" target="new">7538324</a>]<p>5. O'Farrell, H.C., Scarsdale, J.N. and Rife, J.P. Crystal structure of KsgA, a universally conserved rRNA adenine dimethyltransferase in <i>Escherichia coli. J. Mol. Biol.</i> 339 (2004) 337-353. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15136037?dopt=Abstract" target="new">15136037</a>]<p>6. Poldermans, B., Roza, L. and Van Knippenberg, P.H. Studies on the function of two adjacent <i>N</i><small><sup>6</sup></small>,<i>N</i><small><sup>6</sup></small>-dimethyladenosines near the 3' end of 16 S ribosomal RNA of <i>Escherichia coli</i>. III. Purification and  properties of the methylating enzyme and methylase-30 S interactions. <i>J. Biol. Chem.</i> 254 (1979) 9094-9100. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/383712?dopt=Abstract" target="new">383712</a>]<p>7. Demirci, H., Belardinelli, R., Seri, E., Gregory, S.T., Gualerzi, C., Dahlberg, A.E. and Jogl, G. Structural rearrangements in the active site of the <i>Thermus thermophilus</i> 16S rRNA methyltransferase KsgA in a binary complex with 5'-methylthioadenosine. <i>J. Mol. Biol.</i> 388 (2009) 271-282. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19285505?dopt=Abstract" target="new">19285505</a>]<p>8. Tu, C., Tropea, J.E., Austin, B.P., Court, D.L., Waugh, D.S. and Ji, X. Structural basis for binding of RNA and cofactor by a KsgA methyltransferase. <i>Structure</i> 17 (2009) 374-385. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19278652?dopt=Abstract" target="new">19278652</a>]<p><center><small>[EC 2.1.1.182 created 1976 as EC 2.1.1.48, part transferred 2010 to EC 2.1.1.182]</small></center><p><hr>Return to <a href="../1/">EC 2.1.1 home page</a><br>Return to <a href="../">EC 2.1 home page</a><br>Return to <a href="../../">EC 2 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>