<html><head><title>EC 4.3.1.24</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC4/3/1/24.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 4.3.1.24</b></h1></center><b>Accepted name:</b> phenylalanine ammonia-lyase<p><b>Reaction:</b> <small>L</small>-phenylalanine = <i>trans-</i>cinnamate + NH<small><sub>3</sub></small><p>For diagram of reaction <a href="../../../reaction/phenol/chalcone.html">click here</a><p><b>Other name(s):</b> phenylalanine deaminase; phenylalanine ammonium-lyase; PAL; <small>L</small>-phenylalanine ammonia-lyase; Phe ammonia-lyase<p><b>Systematic name:</b> <small>L</small>-phenylalanine ammonia-lyase (<i>trans-</i>cinnamate-forming)<p><b>Comments:</b> This enzyme is a member of the aromatic amino acid lyase family, other members of which are <a href="3.html">EC 4.3.1.3</a> (histidine ammonia-lyase) and <a href="23.html">EC 4.3.1.23</a> (tyrosine ammonia-lyase) and <a href="25.html">EC 4.3.1.25</a> (phenylalanine/tyrosine ammonia-lyase). The enzyme contains the cofactor 3,5-dihydro-5-methylidene-4<i>H-</i>imidazol-4-one (MIO), which is common to this family [3]. This unique cofactor is formed autocatalytically by cyclization and dehydration of the three amino-acid residues alanine, serine and glycine [9]. The enzyme from some species is highly specific for phenylalanine [7,8].<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=4.3.1.24" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/4.3.1.24" target="new">EXPASY</a>, <a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:4.3.1.24" target="new">KEGG</a>, <a href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-4.3.1.24" target="new">Metacyc</a>,<a target="new" href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=4.3.1.24">PDB</a>, CAS registry number: 9024-28-6<p><b>References:</b><p>1. Koukol, J. and Conn, E.E. The metabolism of aromatic compounds in higher plants. IV. Purification and properties of the phenylalanine deaminase of <i>Hordeum vulgare. J. Biol. Chem.</i> 236 (1961) 2692-2698. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/14458851?dopt=Abstract" target="new">14458851</a>]<p>2. Young, M.R. and Neish, A.C. Properties of the ammonia-lyases deaminating phenylalanine and related compounds in <i>Triticum sestivum</i> and <i>Pteridium aquilinum. Phytochemistry</i> 5 (1966) 1121-1132.<p>3. Louie, G.V., Bowman, M.E., Moffitt, M.C., Baiga, T.J., Moore, B.S. and Noel, J.P. Structural determinants and modulation of substrate specificity in phenylalanine-tyrosine ammonia-lyases. <i>Chem. Biol.</i> 13 (2006) 1327-1338. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17185228?dopt=Abstract" target="new">17185228</a>]<p>4. Calabrese, J.C., Jordan, D.B., Boodhoo, A., Sariaslani, S. and Vannelli, T. Crystal structure of phenylalanine ammonia lyase: multiple helix dipoles implicated in catalysis. <i>Biochemistry</i> 43 (2004) 11403-11416. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15350127?dopt=Abstract" target="new">15350127</a>]<p>5. Ritter, H. and Schulz, G.E. Structural basis for the entrance into the phenylpropanoid metabolism catalyzed by phenylalanine ammonia-lyase. <i>Plant Cell</i> 16 (2004) 3426-3436. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15548745?dopt=Abstract" target="new">15548745</a>]<p>6. Watts, K.T., Mijts, B.N., Lee, P.C., Manning, A.J. and Schmidt-Dannert, C. Discovery of a substrate selectivity switch in tyrosine ammonia-lyase, a member of the aromatic amino acid lyase family. <i>Chem. Biol.</i> 13 (2006) 1317-1326. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17185227?dopt=Abstract" target="new">17185227</a>]<p>7. Appert, C., Logemann, E., Hahlbrock, K., Schmid, J. and Amrhein, N. Structural and catalytic properties of the four phenylalanine ammonia-lyase isoenzymes from parsley (<i>Petroselinum crispum</i> Nym.). <i>Eur. J. Biochem.</i> 225 (1994) 491-499. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7925471?dopt=Abstract" target="new">7925471</a>]<p>8. Cochrane, F.C., Davin, L.B. and Lewis, N.G. The <i>Arabidopsis</i> phenylalanine ammonia lyase gene family: kinetic characterization of the four PAL isoforms. <i>Phytochemistry</i> 65 (2004) 1557-1564. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15276452?dopt=Abstract" target="new">15276452</a>]<p>9. Schwede, T.F., R&eacute;tey, J. and Schulz, G.E. Crystal structure of histidine ammonia-lyase revealing a novel polypeptide modification as the catalytic electrophile. <i>Biochemistry</i> 38 (1999) 5355-5361. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10220322?dopt=Abstract" target="new">10220322</a>]<p><center><small>[EC 4.3.1.24 created 2008 (EC 4.3.1.5 created 1965, part-incorporated 2008)]</small></center><p><hr>Return to <a href="../1/">EC 4.3.1 home page</a><br>Return to <a href="../">EC 4.3 home page</a><br>Return to <a href="../../">EC 4 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>