<html><head><title>EC 1.11.2.4</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC1/11/2/4.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 1.11.2.4</b></h1></center><b>Accepted name:</b> fatty-acid peroxygenase<p><b>Reaction:</b> fatty acid + H<small><sub>2</sub></small>O<small><sub>2</sub></small> = 3- or 2-hydroxy fatty acid + H<small><sub>2</sub></small>O<p><b>Other name(s):</b> fatty acid hydroxylase; <i>P</i><small><sub>450</sub></small> peroxygenase; CYP152A1; <i>P</i><small><sub>450</sub></small>BS; <i>P</i><small><sub>450</sub></small>SP&alpha;<p><b>Systematic name:</b> fatty acid:hydroperoxide oxidoreductase (RH-hydroxylating)<p><b>Comments:</b> A cytosolic heme-thiolate protein with sequence homology to <i>P</i><small><sub>450</sub></small> monooxygenases. Unlike the latter, it needs neither NAD(P)H, dioxygen nor specific reductases for function. Enzymes of this type are produced by bacteria (e.g. <i>Sphingomonas paucimobilis. Bacillus subtilis</i>). Catalytic turnover rates are high compared with those of monooxygenation reactions as well as peroxide shunt reactions catalysed by the common <i>P</i><small><sub>450</sub></small>s. A model substrate is myristate, but other saturated and unsaturated fatty acids are also hydroxylated. Oxidizes the peroxidase substrate 3,3',5,5'-tetramethylbenzidine (TMB) and peroxygenates aromatic substrates in a fatty-acid-dependent reaction.<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=1.11.2.4" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/1.11.2.4" target="new">EXPASY</a>, <a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:1.11.2.4" target="new">KEGG</a>, <a target="new" href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-1.11.2.4">Metacyc</a>,<a href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=1.11.2.4" target="new">PDB</a>, CAS registry number: <p><b>References:</b><p>1. Matsunaga, I., Yamada, M., Kusunose, E., Nishiuchi, Y., Yano, I. and Ichihara, K. Direct involvement of hydrogen peroxide in bacterial &alpha;-hydroxylation of fatty acid. <i>FEBS Lett.</i> 386 (1996) 252-254. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/8647293?dopt=Abstract" target="new">8647293</a>]<p>2. Matsunaga, I., Yamada, M., Kusunose, E., Miki, T. and Ichihara, K. Further characterization of hydrogen peroxide-dependent fatty acid &alpha;-hydroxylase from <i>Sphingomonas paucimobilis. J. Biochem.</i> 124 (1998) 105-110. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/9644252?dopt=Abstract" target="new">9644252</a>]<p>3. Matsunaga, I., Ueda, A., Fujiwara, N., Sumimoto, T. and Ichihara, K. Characterization of the <i>ybdT</i> gene product of <i>Bacillus subtilis</i>: novel fatty acid &beta;-hydroxylating cytochrome <i>P</i><small><sub>450</sub></small>. <i>Lipids</i> 34 (1999) 841-846. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10529095?dopt=Abstract" target="new">10529095</a>]<p>4. Imai, Y., Matsunaga, I., Kusunose, E. and Ichihara, K. Unique heme environment at the putative distal region of hydrogen peroxide-dependent fatty acid &alpha;-hydroxylase from <i>Sphingomonas paucimobilis</i> (peroxygenase <i>P</i><small><sub>450</sub></small>SP&alpha;). <i>J. Biochem.</i> 128 (2000) 189-194. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10920253?dopt=Abstract" target="new">10920253</a>]<p>5. Matsunaga, I., Yamada, A., Lee, D.S., Obayashi, E., Fujiwara, N., Kobayashi, K., Ogura, H. and Shiro, Y. Enzymatic reaction of hydrogen peroxide-dependent peroxygenase cytochrome <i>P</i><small><sub>450</sub></small>s: kinetic deuterium isotope effects and analyses by resonance Raman spectroscopy. <i>Biochemistry</i> 41 (2002) 1886-1892. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11827534?dopt=Abstract" target="new">11827534</a>]<p>6. Lee, D.S., Yamada, A., Sugimoto, H., Matsunaga, I., Ogura, H., Ichihara, K., Adachi, S., Park, S.Y. and Shiro, Y. Substrate recognition and molecular mechanism of fatty acid hydroxylation by cytochrome <i>P</i><small><sub>450</sub></small> from <i>Bacillus subtilis</i>. Crystallographic, spectroscopic, and mutational studies. <i>J. Biol. Chem.</i> 278 (2003) 9761-9767. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12519760?dopt=Abstract" target="new">12519760</a>]<p>7. Matsunaga, I. and Shiro, Y. Peroxide-utilizing biocatalysts: structural and functional diversity of heme-containing enzymes. <i>Curr. Opin. Chem. Biol.</i> 8 (2004) 127-132. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15062772?dopt=Abstract" target="new">15062772</a>]<p>8. Shoji, O., Wiese, C., Fujishiro, T., Shirataki, C., Wunsch, B. and Watanabe, Y. Aromatic C-H bond hydroxylation by <i>P</i><small><sub>450</sub></small> peroxygenases: a facile colorimetric assay for monooxygenation activities of enzymes based on Russig&#8217;s blue formation. <i>J. Biol. Inorg. Chem.</i> 15 (2010) 1109-1115. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20490877?dopt=Abstract" target="new">20490877</a>]<p><center><small>[EC 1.11.2.4 created 2011]</small></center><p><hr>Return to <a href="../2/">EC 1.11.2 home page</a><br>Return to <a href="../">EC 1.11 home page</a><br>Return to <a href="../../">EC 1 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>