<html><head><title>EC 4.1.1.77</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC4/1/1/77.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 4.1.1.77</b></h1></center><b>Accepted name:</b> 2-oxo-3-hexenedioate decarboxylase<p><b>Reaction:</b> (3<i>E</i>)-2-oxohex-3-enedioate = 2-oxopent-4-enoate + CO<small><sub>2</sub></small><p>For diagram of reaction <a href="../../../reaction/misc/catechol.html">click here</a>.<p><b>Other name(s):</b> 4-oxalocrotonate carboxy-lyase (misleading); 4-oxalocrotonate decarboxylase (misleading); <i>cnbF</i> (gene name); <i>praD</i> (gene name); <i>amnE</i> (gene name); <i>nbaG</i> (gene name); <i>xylI</i> (gene name)<p><b>Systematic name:</b> (3<i>E</i>)-2-oxohex-3-enedioate carboxy-lyase (2-oxopent-4-enoate-forming)<p><b>Comments:</b> Involved in the meta-cleavage pathway for the degradation of phenols, modified phenols and catechols. The enzyme has been reported to accept multiple tautomeric forms [1-4]. However, careful analysis of the stability of the different tautomers, as well as characterization of the enzyme that produces its substrate, <a href="../../../EC5/3/2/6.html">EC 5.3.2.6</a>, 2-hydroxymuconate tautomerase, showed that the actual substrate for the enzyme is (3<i>E</i>)-2-oxohex-3-enedioate [4].<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=4.1.1.77" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/4.1.1.77" target="new">EXPASY</a>, <a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:4.1.1.77" target="new">KEGG</a>, <a href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-4.1.1.77" target="new">Metacyc</a>,<a target="new" href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=4.1.1.77">PDB</a>, <a href="http://umbbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=4.1.1.77" target="new">UM-BBD</a>, CAS registry number: 37325-55-6<p><b>References:</b><p>1. Shingler, V., Marklund, U., Powlowski, J. Nucleotide sequence and functional analysis of the complete phenol/3,4-dimethylphenol catabolic pathway of <i>Pseudomonas</i> sp. strain CF600. <i>J. Bacteriol.</i> 174 (1992) 711-724. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1732207?dopt=Abstract" target="new">1732207</a>]<p>2. Takenaka, S., Murakami, S., Shinke, R. and Aoki, K. Metabolism of 2-aminophenol by <i>Pseudomonas</i> sp. AP-3: modified meta-cleavage pathway. <i>Arch. Microbiol.</i> 170 (1998) 132-137. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/9683650?dopt=Abstract" target="new">9683650</a>]<p>3. Stanley, T.M., Johnson, W.H., Jr., Burks, E.A., Whitman, C.P., Hwang, C.C. and Cook, P.F. Expression and stereochemical and isotope effect studies of active 4-oxalocrotonate decarboxylase. <i>Biochemistry</i> 39 (2000) 718-726. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10651637?dopt=Abstract" target="new">10651637</a>]<p>4. Wang, S.C., Johnson, W.H., Jr., Czerwinski, R.M., Stamps, S.L. and Whitman, C.P. Kinetic and stereochemical analysis of YwhB, a 4-oxalocrotonate tautomerase homologue in <i>Bacillus subtilis</i>: mechanistic implications for the YwhB- and 4-oxalocrotonate tautomerase-catalyzed reactions. <i>Biochemistry</i> 46 (2007) 11919-11929. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17902707?dopt=Abstract" target="new">17902707</a>]<p>5. Kasai, D., Fujinami, T., Abe, T., Mase, K., Katayama, Y., Fukuda, M. and Masai, E. Uncovering the protocatechuate 2,3-cleavage pathway genes. <i>J. Bacteriol.</i> 191 (2009) 6758-6768. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19717587?dopt=Abstract" target="new">19717587</a>]<p><center><small>[EC 4.1.1.77 created 1999, modified 2011, modified 2012]</small></center><p><hr>Return to <a href="../1/">EC 4.1.1 home page</a><br>Return to <a href="../">EC 4.3 home page</a><br>Return to <a href="../../">EC 4 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>