<html><head><title>EC 3.1.1.96</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC3/1/1/96.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 3.1.1.96</b></h1></center><b>Accepted name:</b> <small>D</small>-aminoacyl-tRNA deacylase<p><b>Reaction:</b> a <small>D</small>-aminoacyl-tRNA + H<small><sub>2</sub></small>O = a <small>D</small>-amino acid + tRNA <p><b>Other name(s):</b> Dtd2; <small>D</small>-Tyr-tRNA(Tyr) deacylase; <small>D</small>-Tyr-tRNA<small><sup>Tyr</sup></small> deacylase; <small>D</small>-tyrosyl-tRNA<small><sup>Tyr</sup></small> aminoacylhydrolase; <i>dtdA</i> (gene name)<p><b>Systematic name:</b> <small>D</small>-aminoacyl-tRNA aminoacylhydrolase<p><b>Comments:</b> The enzyme from <i>Escherichia coli</i> can cleave <small>D</small>-tyrosyl-tRNA<small><sup>Tyr</sup></small>, <small>D</small>-aspartyl-tRNA<small><sup>Asp</sup></small> and <small>D</small>-tryptophanyl-tRNA<small><sup>Trp</sup></small> [1]. Whereas the enzyme from the archaeon <i>Pyrococcus abyssi</i> is a zinc protein, the enzyme from <i>Escherichia coli</i> does not carry any zinc [2].<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=3.1.1.96" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/3.1.1.96" target="new">EXPASY</a>,<a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:3.1.1.96" target="new">KEGG</a>, <a href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-3.1.1.96" target="new">Metacyc</a>,CAS registry number: <p><b>References:</b><p>1. Soutourina, J., Plateau, P. and Blanquet, S. Metabolism of <small>D</small>-aminoacyl-tRNAs in <i>Escherichia coli</i> and <i>Saccharomyces cerevisiae</i> cells. <i>J. Biol. Chem.</i> 275 (2000) 32535-32542. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10918062?dopt=Abstract" target="new">10918062</a>]<p>2. Ferri-Fioni, M.L., Schmitt, E., Soutourina, J., Plateau, P., Mechulam, Y. and Blanquet, S. Structure of crystalline <small>D</small>-Tyr-tRNA(Tyr) deacylase. A representative of a new class of tRNA-dependent hydrolases. <i>J. Biol. Chem.</i> 276 (2001) 47285-47290. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11568181?dopt=Abstract" target="new">11568181</a>]<p>3. Ferri-Fioni, M.L., Fromant, M., Bouin, A.P., Aubard, C., Lazennec, C., Plateau, P. and Blanquet, S. Identification in archaea of a novel <small>D</small>-Tyr-tRNA<small><sup>Tyr</sup></small> deacylase. <i>J. Biol. Chem.</i> 281 (2006) 27575-27585. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16844682?dopt=Abstract" target="new">16844682</a>]<p>4. Wydau, S., Ferri-Fioni, M.L., Blanquet, S. and Plateau, P. GEK1, a gene product of <i>Arabidopsis thaliana</i> involved in ethanol tolerance, is a <small>D</small>-aminoacyl-tRNA deacylase. <i>Nucleic Acids Res.</i> 35 (2007) 930-938. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17251192?dopt=Abstract" target="new">17251192</a>]<p><center><small>[EC 3.1.1.96 created 2014]</small></center><p><hr>Return to <a href="../1/">EC 3.1.1 home page</a><br>Return to <a href="../">EC 3.1 home page</a><br>Return to <a href="../../">EC 3 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>