<html><head><title>EC 3.5.99.2</title></head><base href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC3/5/99/2.html"><body><center><b>IUBMB Enzyme Nomenclature</b><p><h1><b>EC 3.5.99.2</b></h1></center><b>Accepted name:</b> aminopyrimidine aminohydrolase<p><b>Reaction:</b> (1) 4-amino-5-aminomethyl-2-methylpyrimidine + H<small><sub>2</sub></small>O = 4-amino-5-hydroxymethyl-2-methylpyrimidine + NH<small><sub>3</sub></small><br>(2) thiamine + H<small><sub>2</sub></small>O = 4-amino-5-hydroxymethyl-2-methylpyrimidine + 5-(2-hydroxyethyl)-4-methylthiazole<p><b>Other name(s):</b> thiaminase, thiaminase II, <i>tenA</i> (gene name)<p><b>Systematic name:</b> 4-amino-5-aminomethyl-2-methylpyrimidine aminohydrolase<p><b>Comments:</b> Previously known as thiaminase II, this enzyme is involved in the regeneration of the thiamine pyrimidine from degraded products, rather than in thiamine degradation, and participates in thiamine salvage pathways.<p><b>Links to other databases:</b> <a href="http://www.brenda-enzymes.org/enzyme.php?ecno=3.5.99.2" target="new">BRENDA</a>, <a href="http://enzyme.expasy.org/EC/3.5.99.2" target="new">EXPASY</a>, <a href="http://www.genome.ad.jp/dbget-bin/www_bget?ec:3.5.99.2" target="new">KEGG</a>, <a target="new" href="http://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-3.5.99.2">Metacyc</a>,<a href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=3.5.99.2" target="new">PDB</a>, <a href="http://umbbd.ethz.ch/servlets/pageservlet?ptype=e&ECcode=3.5.99.2" target="new">UM-BBD</a>, CAS registry number: 9024-80-0<p><b>References:</b><p>1. Fujita, A., Nose, Y. and Kuratani, K. Second type of bacterial thiaminase. <i>J. Vitaminol. (Kyoto)</i> 1 (1954) 1-7. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/13243520?dopt=Abstract" target="new">13243520</a>]<p>2. Ikehata, H. Purification of thiaminase II. <i>J. Gen. Appl. Microbiol.</i> 6 (1960) 30-39.<p>3. Toms, A.V., Haas, A.L., Park, J.H., Begley, T.P. and Ealick, S.E. Structural characterization of the regulatory proteins TenA and TenI from <i>Bacillus subtilis</i> and identification of TenA as a thiaminase II. <i>Biochemistry</i> 44 (2005) 2319-2329. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15709744?dopt=Abstract" target="new">15709744</a>]<p>4. Benach, J., Edstrom, W.C., Lee, I., Das, K., Cooper, B., Xiao, R., Liu, J., Rost, B., Acton, T.B., Montelione, G.T. and Hunt, J.F. The 2.35 &Aring; structure of the TenA homolog from <i>Pyrococcus furiosus</i> supports an enzymatic function in thiamine metabolism. <i>Acta Crystallogr. D Biol. Crystallogr.</i> 61 (2005) 589-598. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15858269?dopt=Abstract" target="new">15858269</a>]<p>5. Jenkins, A.H., Schyns, G., Potot, S., Sun, G. and Begley, T.P. A new thiamin salvage pathway. <i>Nat. Chem. Biol.</i> 3 (2007) 492-497. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17618314?dopt=Abstract" target="new">17618314</a>]<p>6. Jenkins, A.L., Zhang, Y., Ealick, S.E. and Begley, T.P. Mutagenesis studies on TenA: a thiamin salvage enzyme from <i>Bacillus subtilis. Bioorg. Chem.</i> 36 (2008) 29-32. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/18054064?dopt=Abstract" target="new">18054064</a>]<p>7. French, J.B., Begley, T.P. and Ealick, S.E. Structure of trifunctional THI20 from yeast. <i>Acta Crystallogr. D Biol. Crystallogr.</i> 67 (2011) 784-791. [PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21904031?dopt=Abstract" target="new">21904031</a>]<p><center><small>[EC 3.5.99.2 created 1961, deleted 2011, modified 2011]</small></center><p><hr>Return to <a href="../99/">EC 3.5.99 home page</a><br>Return to <a href="../">EC 3.5 home page</a><br>Return to <a href="../../">EC 3 home page</a><br>Return to <a href="../../../">Enzymes home page</a><br>Return to <a href="../../../../">IUBMB Biochemical Nomenclature home page</a><p></body></html>